app/features/receive/cashu-receive-quote-service.ts

```diff
@@ -13,12 +15,15 @@ import {
 import type { Money } from '~/lib/money';
 import type { CashuAccount } from '../accounts/account';
 import { type CashuCryptography, useCashuCryptography } from '../shared/cashu';
+import { derivePublicKey } from '../shared/cryptography';
 import type { CashuReceiveQuote } from './cashu-receive-quote';
 import {
   type CashuReceiveQuoteRepository,
   useCashuReceiveQuoteRepository,
 } from './cashu-receive-quote-repository';
 
+const BASE_CASHU_LOCKING_DERIVATION_PATH = "m/129372'/0'/0'";
```

Comment:
does this bleong here? will this be used for all cashu locking we will be doing? also can you add some comment or link aobut why this value

Reply:
I see now that you already moved it in the other pr